package.json

```diff
@@ -54,6 +54,7 @@
     "tslib": "^1.6.0",
     "tslint": "^5.3.2",
     "tslint-microsoft-contrib": "^5.0.0",
+    "tsutils": "^1.6.0",
```

Comment:
Travis is failing due to missing peer dependency.  I suspect this might be a transient travis error not allowing the continuation on the missing dependency on npm install but fixing longer term by adding the devDependency